src/App.jsx

```diff
@@ -20,6 +20,9 @@ export function App() {
 	const [listPath, setListPath] = useStateWithStorage(
 		'tcl-shopping-list-path',
 		null,
+		{
+			/* use "MyvBY6DUacb6kpGB6wtE1v5QZjF2/test-list" to render list */
+		},
```

Comment:
How do we use this to render the list?

Reply:
oops sorry, just read the instructions

Reply:
I think the only thing I would change in this comment (so it's clear from the comment itself, for our future selves) would be to let us know to replace null with that string.

Reply:
maybe we should add a comment here that this is needed temporarily but we can remove later?

Reply:
totally agree

Reply:
Thank you! Will remove the console logs and add that comment!